addOns/grpc/src/test/java/org/zaproxy/addon/grpc/internal/ProtoBufMessageEncoderUnitTest.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.grpc.internal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ProtoBufMessageEncoderUnitTest {
+
+    private ProtoBufMessageEncoder encoder;
+
+    @BeforeEach
+    public void setUp() {
+        encoder = new ProtoBufMessageEncoder();
+    }
+
+    @Test
+    void testStartEncodingWithEmptyInput() {
+        String inputString = "";
+        byte[] decodedBytes = org.apache.commons.codec.binary.Base64.decodeBase64(inputString);
+        try {
+            List<String> messageFields = EncoderUtils.parseIntoList(inputString);
+            encoder.encode(messageFields);
+        } catch (Exception e) {
+        }
```

Comment:
If nothing is happening in the catch block why not just have the method indicate it throws?

Reply:
Throughout this test class